src/aind_data_schema_models/mouse_anatomy.py

```diff
@@ -0,0 +1,45 @@
+"""Module for Mouse Anatomy"""
+
+from importlib_resources import files
+from pydantic import BaseModel, ConfigDict, Field
+
+from aind_data_schema_models.registries import RegistryModel, map_registry
+from aind_data_schema_models.utils import create_literal_class, read_csv, one_of_instance
+
+
+class MouseAnatomyModel(BaseModel):
+    """Base model config"""
+
+    model_config = ConfigDict(frozen=True)
+
+    id: int = Field(..., title="Structure EMAPA ID")
+    name: str = Field(..., title="Structure name")
+    registry: RegistryModel = None
```

Comment:
we can hard-code the registry to the EMAPA registry

Reply:
This was confusing to implement, would be good to have an onboarding doc for schemas somewhere where we mention that custom types need to be Annotated to validate properly. The pydantic error you get is mysterious.